Uzduotis10/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Uzduotis10
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            // Liepkite įvesti skaičių.
+            /* Jei skaičius lyginis, tuomet išveskite šį skaičių bei jo penktą laipsnį.
+             * Jei skaičius nesidalina iš 4, tuomet liepkite įvesti antrąjį skaičių. 
+             * Išveskite abiejų skaičių sumą ir sandaugą.
+            */
+
+            Console.Write("Please enter a number: ");
+            double number = double.Parse(Console.ReadLine());
+
+            if (number % 2 == 0)
+            {
+                double fifthPower = Math.Pow(number, 5);
+                Console.WriteLine($"Number: {number}");
+                Console.WriteLine($"Fifth Power: {fifthPower}");
+            }
+            else
+            {
+                if (number % 4 != 0)
```

Comment:
Čia turėtų būti kaip atskira `if` sąlyga o ne `else sąlygos dalis. Be to, jei jau taip rašytumėte tai logiškiau daryti `else if` bloką nei `else` bloką ir jo viduje `if` bloką.